lib/rubber/cloud/datto.rb

```diff
@@ -0,0 +1,97 @@
+require 'rubber/cloud/generic'
+require 'httparty'
+
+module Rubber
+  module Cloud
+    class Datto < Base
+      def initialize(env, capistrano)
+        super(env, capistrano)
```

Comment:
Why duplicate this method? Also, if you want to invoke super w/ the normal args, you don't need to pass it any args:
```ruby
super
```